### PR TITLE
Chapter 10

### PR DESCRIPTION
**Pull request by heimrih**
Chapter 10
Updating, showing, and deleting users

### CODE REVIEW

**Comment by heimrih** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -21,8 +21,14 @@ gem "sass-rails", ">= 6"
 
 gem "bootstrap-sass", "3.4.1"
 
+gem "bootstrap-will_paginate", "1.0.0"
```
Comment (ngocvt-0484) [inline Gemfile]: remove it, are u using this gem ?
if remove it, the application work ok? right?

Comment (heimrih) [reply Gemfile]: image
this error come out when i delete bootstrap -will paginate and run bundle install and restart server. How do i fix it with kaminari?

Comment (heimrih) [reply Gemfile]: nevermind. i solved.

**Comment by heimrih** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**